webi-essentials/README.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Webi Essentials
+homepage: https://webinstall.dev/webi-essentials
+tagline: |
+  The Webi Essentials are curl, git, tar, wget, xz, zip, & webi shell integrations
+---
+
+### Cheat Sheet
+
+> Even Webi needs a place to start.
+
+This installs the tools that are commonly needed to download and unpack various
+packages, using the native package manager.
+
+**This requires root or `sudo`**
+
+### Linux (Debian, Ubuntu)
```

Comment:
Are these the manual installation instructions? If so, would it be better to heading that off similarly to other packages?

Reply:
What do you mean?

Reply:
I understand. Will update.